maestro-android/src/main/java/dev/mobile/maestro/receivers/LocaleSettingReceiver.kt

```diff
@@ -0,0 +1,105 @@
+package dev.mobile.maestro.receivers
+
+import android.app.Activity
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import dev.mobile.maestro.handlers.LocaleSettingHandler
+import org.apache.commons.lang3.LocaleUtils
+import java.util.*
+
+class LocaleSettingReceiver : BroadcastReceiver(), HasAction {
```

Comment:
I am a little confused about how are we planning to ship this:

* This PR doesn't invoke this broadcast receiver
* Where are we planning to send the broadcast from
* How is this going to wire with Maestro Client?

Reply:
> Where are we planning to send the broadcast from
coming in a separate PR , this is independent logic, so can be merged separately. small PRs are easier to review imo

> How is this going to wire with Maestro Client?
adb call will be part of DeviceService, integration PR is coming soon

Reply:
>coming in a separate PR , this is independent logic, so can be merged separately. small PRs are easier to review imo

I agree. But how is this going to work end to end? Is there a doc somewhere?

Reply:
we decided not to write an RFC for this feature, since it looked rather small in the beginning, but looking retrospectively now it made sense to write an RFC